### PR TITLE
test title

### PR DESCRIPTION
**Pull request by shuvro-baset**
23.8 review, please

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -341,8 +341,20 @@ <h1>DOM Section</h1>
   </section>
 
   <section class="cards">
```
Comment (muhammadhafijur) [inline index.html:343]: add a html comment with module name

Comment (shuvro-baset) [reply index.html:343]: okay got it

**Review by shuvro-baset — COMMENTED** [review]
check 24.6

**Comment by shuvro-baset** [comment]
adding eventListener module 24.6

**Comment by muhammadhafijur** [comment]
add file in data js

**Pull request closed without merging**